sql/core/src/test/scala/org/apache/spark/sql/ColumnExpressionSuite.scala

```diff
@@ -923,4 +923,452 @@ class ColumnExpressionSuite extends QueryTest with SharedSparkSession {
     val inSet = InSet(Literal("a"), Set("a", "b").map(UTF8String.fromString))
     assert(inSet.sql === "('a' IN ('a', 'b'))")
   }
+
+  {
```

Comment:
do we need to wrap the code with `{...}`?

Reply:
No, it's not necessary. I was doing it because I did not see any reason for the withField specific data and methods to be accessible outside of the scope.

Reply:
Ah it's OK to have some private methods that are only called by a few test cases. Too many indentations can make the test hard to read.

Reply:
fair enough, I've made the change.